examples/00-load/create-struss.py

```diff
@@ -0,0 +1,60 @@
+"""
+.. _ref_create_explicit_structured_grid:
+
+Plot Truss-like FEA Solution with Cylinders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Plot connections between points in 3D as cylinders, colored by scalars.
+
+
+"""
+
+import numpy as np
+import pyvista
+
+###############################################################################
+# Define the points and elements of the truss.  Call them ``nodes``
+# here as it comes from finite element analysis.
+
+nodes = [
+    [0.0, 0.0, 0.0],
+    [0.0, 1.0, 0.0],
+    [4.0, 3.0, 0.0],
+    [4.0, 0.0, 0.0],
+    [0.0, 1.0, 2.0],
+    [4.0, 1.0, 2.0],
+    [4.0, 3.0, 2.0]
+    ]
+
+
+edges = np.array([
+    [0, 4],
+    [1, 4],
+    [3, 4],
+    [5, 4],
+    [6, 4],
+    [3, 5],
+    [2, 5],
+    [5, 6],
+    [2, 6],
+    ])
+
+# We must "pad" the edges to indicate to vtk how many points per edge
```

Comment:
points or nodes here?  I'm not sure.  It might be a good place to point out quickly why the padding is necessary, or maybe link to the PolyData class where it is explained more?